misk-zookeeper/src/main/kotlin/misk/clustering/zookeeper/CuratorFrameworkProvider.kt

```diff
@@ -12,6 +12,15 @@ internal class CuratorFrameworkProvider @Inject internal constructor(
 ) : Provider<CuratorFramework> {
 
   override fun get(): CuratorFramework {
+    if (config.cert_store != null && config.trust_store != null) {
+      System.setProperty("zookeeper.clientCnxnSocket", "org.apache.zookeeper.ClientCnxnSocketNetty")
+      System.setProperty("zookeeper.client.secure", "true")
+      System.setProperty("zookeeper.ssl.keyStore.location", config.cert_store.resource)
+      System.setProperty("zookeeper.ssl.keyStore.password", config.cert_store.passphrase ?: "changeit")
+      System.setProperty("zookeeper.ssl.trustStore.location", config.trust_store.resource)
+      System.setProperty("zookeeper.ssl.trustStore.password", config.trust_store.passphrase ?: "changeit")
```

Comment:
make this default value on the Config object instead?

Reply:
well we could have a `JksTrustStoreConfig` that provides this default. anywhere we load a jks trust store is going to have this logic

Reply:
Eh, feels overkill and then odd that we have a JksTrustStore config and then a general TrustStore config where you set a format.